swift/ql/src/queries/Security/CWE-1204/StaticInitializationVector.qhelp

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+
+<overview>
+  <p>When a cipher is used in certain modes (such as CBC or GCM), it requires an initialization vector (IV). Under the same secret key, IVs should be unique and ideally unpredictable. If the same IV is used with the same secret key, then the same plaintext results in the same ciphertext. This behavior may enable an attacker to learn if the same data pieces are transferred or stored, or help the attacker run a dictionary attack.</p>
+  <p>In particular, if the IV is hardcoded or constant, an attacker may just lookup potential keys in a dictionary, then concatenate those with the hardcoded or constant IV rather than trying to discover the entire encryption key.</p>
```

Comment:
`lookup` is a noun, `to look up` is the verb, to I'd be tempted to use `an attacker may just look up`
However, I am not familiar with this particular use (this may be widely used in the developer world) so feel free to ignore this comment if you don't agree.

Reply:
Sure. I've changed that to the suggested wording.